nncf/common/quantization/structs.py

```diff
@@ -213,6 +213,7 @@ def is_config_compatible(self, qconfig: QuantizerConfig) -> bool:
                 qconf_attr_value = getattr(qconfig, attr_name)
                 if qconf_attr_value != constraint:
                     is_compatible = False
```

Comment:
What do you think about `return False` here?

```
def is_config_compatible(self, qconfig: QuantizerConfig) -> bool:
        for attr_name, constraint in self.qconf_attr_vs_constraint_dict.items():
            if constraint is not None:
                qconf_attr_value = getattr(qconfig, attr_name)
                if qconf_attr_value != constraint:
                    return False
        return True
```

Reply:
> What do you think about `return False` here?
> 
> ```
> def is_config_compatible(self, qconfig: QuantizerConfig) -> bool:
>         for attr_name, constraint in self.qconf_attr_vs_constraint_dict.items():
>             if constraint is not None:
>                 qconf_attr_value = getattr(qconfig, attr_name)
>                 if qconf_attr_value != constraint:
>                     return False
>         return True
> ```

I think it should be avoided, and the reason is the developer might accidentally forget to return a value from a function. In this case, Python will return None. This can cause subtle bugs that can be difficult to debug.

Reply:
> What do you think about `return False` here?
> 
> ```
> def is_config_compatible(self, qconfig: QuantizerConfig) -> bool:
>         for attr_name, constraint in self.qconf_attr_vs_constraint_dict.items():
>             if constraint is not None:
>                 qconf_attr_value = getattr(qconfig, attr_name)
>                 if qconf_attr_value != constraint:
>                     return False
>         return True
> ```

Hi @alexsu52 thanks for your review. This looks good to me.